### ISSUE TITLE
SVN keyword exansion messed up the User's guide section on version manangement

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/Modelica#299]
**Reported by dietmarw on 26 Feb 2010 08:47 UTC**
In the `Modelica.UsersGuide.ReleaseNotes.VersionManagement` contains information on how to use the different version annotations. The problem is that 
```
revisionId   = \"$Id::                                      $\")
```
will be picked up by the Subversion (or CVS) and a new (perhaps unintended) "Id stamp" is created. This should be improved and the documentation bit should be static and not be changed by the version control system.

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/299

**Comment by modelica-trac-importer** [comment]
**Comment by dietmarw on 26 Feb 2010 08:49 UTC**
This can be fixed by adding a [Zero width space](http://en.wikipedia.org/wiki/Zero-width_space) character into the `revisonId` string so that Subversion and CVS will not detect this as a keyword to react on it.

**Comment by modelica-trac-importer** [comment]
**Comment by dietmarw on 26 Feb 2010 08:59 UTC**
Fixed in 8ec835c45b09af678a00948af9c30f8ba0ddd81f.
